src/Razor/src/Microsoft.AspNetCore.Razor.OmniSharpPlugin.StrongNamed/OmniSharpProjectWorkspaceStateGenerator.cs

```diff
@@ -12,11 +10,6 @@ namespace Microsoft.AspNetCore.Razor.OmniSharpPlugin.StrongNamed;
 
 public class OmniSharpProjectWorkspaceStateGenerator : IOmniSharpProjectSnapshotManagerChangeTrigger
 {
-    // Internal for testing
-    internal OmniSharpProjectWorkspaceStateGenerator()
```

Comment:
This wasn't actually used, and it messed with `InternalWorkspaceStateGenerator`'s nullability.